### DISCUSSION TITLE
More information for the Error Message

### DISCUSSION THREAD

**Discussion by rappen** [rappen/FetchXMLBuilder#617] [Ideas]
After adding a new feature for Error Information - a bit smarter. #613 
But what more info do we need?
Comment here!

This is what we have...
![image](https://user-images.githubusercontent.com/2572253/149626378-385f5f85-b56a-442a-ae9f-3375b038b411.png)

**Comment by rafek1241** [comment]
Button with `Open issue in repository`.

**Reply by rappen** [reply]
It's there!
Update to latest release (1.2022.3.1) and you have these buttons:
![image](https://user-images.githubusercontent.com/2572253/156576337-c12c1028-a499-41f3-a776-a1d4f67e8301.png)
